services/core-api/app/api/mines/permits/permit_extraction/create_permit_conditions.py

```diff
@@ -87,7 +87,8 @@ def create_permit_conditions_from_task(task: PermitExtractionTask):
         return True
 
     except Exception as e:
-        current_app.logger.error("Failed to create permit conditions from task", e)
+        current_app.logger.error("Failed to create permit conditions from task")
+        current_app.logger.error(e)
```

Comment:
every time that the test hit this error (I think about 4 tests) it would output as "TypeError: not all arguments converted during string formatting" rather than the actual error, which was not very helpful, so I separated it out into its own line.